README.md

```diff
@@ -21,3 +26,83 @@ cloudformation yaml files with [pre-commit](https://pre-commit.com).
 Please install pre-commit. Once installed, the file validations will
 automatically run on every commit. Alternatively you can manually
 execute the validations by running `pre-commit run --all-files`.
+
+## Working with the Cluster
+
+### Reaching the Jumpbox
+In the AWS Console, find the jumpbox instance, and ssh it to using the PEM
+that was specified in the stack creation. For example,
+`ssh -i "toil.pem" ubuntu@10.23.95.18`.
+
+### Reaching the Leader
+Once you're on the Jumpbox, use this command to reach the cluster's leader,
+where zone is the zone where you created the jumpbox and leader and you
+specify the cluster name, for example, "rna-seq-reprocessing-toil-cluster-v001":
+`sudo toil ssh-cluster --zone us-east-1c rna-seq-reprocessing-toil-cluster-v001`
```

Comment:
how do you know which zone?  where can i get that info from?

Reply:
"zone is the zone where you created the jumpbox and leader" <- this will be clear from parameters that are set during provisioning